### PR TITLE
[zh] Update access-cluster-services.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh] Update access-cluster-services.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/access-cluster-services.md [hunk 1]
```diff
@@ -160,6 +170,15 @@ If you haven't specified a name for your port, you don't have to specify *port_n
 `http://`*`kubernetes_master_address`*`/api/v1/namespaces/`*`namespace_name`*`/services/`*`service_name[:port_name]`*`/proxy`
 
 如果还没有为你的端口指定名称，你可以不用在 URL 中指定 *port_name*。
+对于命名和未命名端口，您还可以使用端口号代替 *port_name*。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/access-cluster-services.md:173]:
```suggestion
对于命名和未命名端口，你还可以使用端口号代替 *port_name*。
```

File: content/zh/docs/tasks/administer-cluster/access-cluster-services.md [hunk 2]
```diff
@@ -160,6 +170,15 @@ If you haven't specified a name for your port, you don't have to specify *port_n
...
+
+默认情况下，API 服务器使用 HTTP 代理您的服务。 要使用 HTTPS，请在服务名称前加上 `https:`：
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/access-cluster-services.md:175]:
```suggestion
默认情况下，API 服务器使用 HTTP 为你的服务提供代理。要使用 HTTPS，请在服务名称前加上 `https:`：
```

Comment (my-git9) [reply content/zh/docs/tasks/administer-cluster/access-cluster-services.md]: Modified,thanks for your review

**Pull request merged**